app/ui/src/app/import-export/import/import.component.scss

```diff
@@ -0,0 +1,25 @@
+:host {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  overflow: visible;
+}
+
+syndesis-list-toolbar {
+  flex: 0 0 auto;
+}
+
+syndesis-loading {
```

Comment:
Isn't this piece of SASS used elsewhere? If so, perhaps it would be a good idea to promote it to `src/scss/components/_loading.scss`